examples/.markdownlint.yaml

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
```

Comment:
Am I missing something or is the general index readme for the examples missing? Is that intentional?

Reply:
That is intentional. With the discussion around a master README, I realized it's better to bring the README in a separate PR.